readme/skills.md

```diff
@@ -0,0 +1,126 @@
+# Minimum Skillset for Using BLT
+BLT is a tool that reaches far beyond Drupal development, and because of this developers planning on using it should working knowledge of the following technologies in order to be successful.
+
+See the [System Requirements](../INSTALL.md) for installing / using BLT. It is strongly urged that you use a Mac for local development, although certain versions of Linux and Windows 10 can also be used. Other operating systems ***should not*** be used for BLT development.
+ 
+## Enterprise Web Development
+### Virtual Environment
+#### [DrupalVM](https://www.drupalvm.com/)
+This project aims to make spinning up a simple local Drupal test/development environment incredibly quick and easy, and to introduce new developers to the wonderful world of Drupal development on local virtual machines (instead of crufty old MAMP/WAMP-based development).
```

Comment:
We don't require that people use VM with BLT. In fact, I use MAMP :)

I'd remove this or note that it isn't required.